packages/data-service-generator/src/tests/mock-data-plugin-installations.ts

```diff
@@ -9,15 +9,32 @@ export const plugins: Record<string, PluginInstallation> = Object.freeze({
     pluginId: "broker-kafka",
     settings: {},
   },
-  jwtAuth: {
+  authCore: {
+    id: "auth-core",
+    npm: "@amplication/plugin-auth-core",
+    enabled: true,
+    version: "latest",
+    pluginId: "auth-core",
+  },
+  authJwt: {
     id: "clb3p3uxx009bjn01zfbim7p1",
     npm: "@amplication/plugin-auth-jwt",
     enabled: true,
     version: "latest",
     pluginId: "auth-jwt",
     settings: {},
   },
-  postgresPlugin: {
+  grpc: {
+    id: "transport-grpc",
+    npm: "@amplication/plugin-transport-grpc",
+    enabled: true,
+    version: "latest",
+    pluginId: "transport-grpc",
+    configurations: {
+      generateGRPC: "true",
+    },
+  },
```

Comment:
Do we want specific plugins here? Or the default plugins?

Reply:
we wanted consolidate the mock data for plugin installations in our snapshot tests.